releases.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Releases
+summary: Release notes for older versions of CockroachDB.
+toc: false
+---
+
+<style>
+.release-table .fa {
+    margin-right: 2px;
+}
+
+.os-release-link {
+    margin-right: 20px;
+}
+
+.os-release-link:last-child {
+    margin-right: 0;
+}
+
+.os-release-cell {
+    padding-left: 30px;
+    padding-right: 30px;
+    white-space: nowrap;
+}
+
+.latest {
+    background-color: #fafdf6;
+}
+
+.badge {
+    background-color: #4eb21d;
+    font-size: 10px;
+    text-transform: uppercase;
+    vertical-align: middle;
+}
+</style>
+
+After downloading your desired release, learn how to [Install CockroachDB](install-cockroachdb.html).
+
+<table class="release-table">
+<thead>
+<tr>
+  <td>Version</td>
```

Comment:
<td>Version &amp; Release Notes</td> just so it's clear what's at the other end of the link